src/index.html

```diff
@@ -7,8 +7,85 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        class="logo" 
+        href="http://localhost:8080/"
+      >
+        <img src="/images/logo.png" alt="Logo">
+      </a>
+      <nav class="nav">
```

Comment:
Add spaces where needed
```suggestion
      </a>
      
      <nav class="nav">
```